### ISSUE TITLE
Is pretrained_word_embedding

### ISSUE THREAD

**Issue by cyb70289** [fchollet/keras#6685]
Below document says example pretrained_word_embeddings.py reaches above 90% accuracy after two epochs. But I can only reach about 30% accuracy. Is the document wrong? Or I missed something? I'm using latest keras(2.0.4) with tensorflow(1.1.0). I downloaded required files and run the example without any change.
https://blog.keras.io/using-pre-trained-word-embeddings-in-a-keras-model.html
